packages/cli/create-strapi-app/src/cloud.ts

```diff
@@ -0,0 +1,97 @@
+import inquirer from 'inquirer';
+import { resolve } from 'node:path';
+import { cli as cloudCli, services as cloudServices } from '@strapi/cloud-cli';
+import parseToChalk from './utils/parse-to-chalk';
+
+interface CloudError {
+  response: {
+    status: number;
+    data: string | object;
+  };
+}
+
+function assertCloudError(e: unknown): asserts e is CloudError {
+  if ((e as CloudError).response === undefined) {
+    throw Error('Expected CloudError');
+  }
+}
+
+export async function handleCloudProject(projectName: string): Promise<void> {
+  let cloudApiService = await cloudServices.cloudApiFactory();
```

Comment:
There are several awaits here that aren't needed

Reply:
What do you mean? cloudServices.cloudApiFactory() is returning a promise.

Reply:
Don't mind me, my editor was messed up 🤦